src/modules/CreateDrops/DropSummary.tsx

```diff
@@ -26,14 +26,41 @@ export const DropSummary = ({
   const { onPrevious } = useDropFlowContext();
   const { costsData, confirmationText, totalCost } = paymentData;
   const { isOpen, onOpen } = useDisclosure();
-  const summaryItems = summaryData.map((item) => (
-    <Box key={item.name} mb="5">
-      <Text fontWeight="medium">{item.name}</Text>
-      <Text fontSize={{ base: 'md', md: 'lg' }} mt="6px">
-        {item.value}
-      </Text>
-    </Box>
-  ));
+  const summaryItems = summaryData.map((item) => {
```

Comment:
nice work on the switch case! i think we should create component for each type of input, otherwise the switch case could be quite long.

```
switch (item.type) {
   case 'image':
      return <SummaryItemImage {...} />
   ....
```

what do you think?

Reply:
sounds good!